app/controllers/api/v1/identifier_types_controller.rb

```diff
@@ -1,6 +1,6 @@
 class Api::V1::IdentifierTypesController < AuthenticatedApiController
   def create
-    identifier_type = IdentifierType.create_with(owner_app: current_app).find_or_initialize_by(create_params)
+    identifier_type = IdentifierType.find_or_initialize_by(create_params.merge(owner_app: current_app))
```

Comment:
Only thing here is that it's gonna start subtly acting wonky if someone adds additional, non-uniquely-identifying params to `create_params`. Maybe this can be expressed like so?:

```ruby
identifier_type = IdentifierType.find_or_initialize_by(owner_app: current_app, name: create_params[:name])
```

Reply:
But I guess it's not really different from the way it was expressed before, so take it or leave it. 🤷‍♂️

Reply:
Yeah, in this case, I made it slightly tighter than the previous iteration in that it won't tell you it's OK if you try to touch another app's identifier_type. This definitely isn't as resilient to future change of this controller as it could be (if I wanted that I'd probably build a migration model for it), but it's also the least used feature of testtrack. Important, but mostly only used when you first set it up for an app ecosystem.